runbot_travis2docker/models/runbot_repo.py

```diff
@@ -40,3 +40,4 @@ def weblate_validation(self):
             json = response.json()
             if 'projects' not in json:
                 raise ValidationError(_('Response json bad formated'))
+        raise Warning(_('Connection with weblate successful'))
```

Comment:
A raise for a positive message I think so is not a good idea
@JulioSerna @hugho-ad 
what do you think?